src/main/java/uk/gov/hmcts/reform/civil/helpers/judgmentsonline/JudgmentsOnlineHelper.java

```diff
@@ -67,4 +72,69 @@ public static List<String> validateMidCallbackData(CaseData caseData) {
         return errors;
     }
 
+    public static boolean isNonDivergentForDJ(CaseData caseData) {
+        return  MultiPartyScenario.isOneVOne(caseData)
+            || MultiPartyScenario.isTwoVOne(caseData)
+            || (ofNullable(caseData.getRespondent2()).isPresent()
+            && ofNullable(caseData.getDefendantDetailsSpec()).isPresent()
+            && ofNullable(caseData.getDefendantDetailsSpec().getValue()).isPresent()
+            && caseData.getDefendantDetailsSpec().getValue().getLabel().startsWith("Both"));
+    }
+
+    public static boolean isNonDivergentForJBA(CaseData caseData) {
+        return  MultiPartyScenario.isOneVOne(caseData)
+            || MultiPartyScenario.isTwoVOne(caseData)
```

Comment:
Currently we are routing the claims to Non-Divergent flow of JBA to keep them Online(ALL_FINAL_ORDERS_ISSUED) state only if they are LRvLiPOnevOne, All other claims would go offline. Should we check for 2v1 as well?

Reply:
No that is diffrent requirment related to state. This method only is used for marking Judgment active or not.

Reply:
ok. Fine..